community/cypher/docs/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -195,9 +195,9 @@ class QueryPlanTest extends DocumentingTestBase {
       title = "Cartesian Product",
       text =
         """Produces a cross product of the two inputs -- each row coming from the left child will be matched with all the rows from the right child operator.
-          |The following query will join all the people with all the locations and return the cartesian product of the nodes with those labels.
+          |The following query will join all the people with all the teams and return the cartesian product of the nodes with those labels.
```

Comment:
My vote is to re-write the entire line. I only changed locations->teams to get the cartesian product to look better. The previous 5*1=5 was not informative. I suggest I write new text now too.